README.md

```diff
@@ -64,16 +64,19 @@ Creating a text message:
 
 Where:
 
-* "mobile-number" is the mobile phone number to deliver to
+* `mobile-number` is the mobile phone number to deliver to
     * Only UK mobiles are supported
     * Must start with +44
     * Must not have leading zero
     * Must not have any whitespace, punctuation etc.
     * valid format is +447777111222
-    
-* "template_id" is the template to send
+
+* `template_id` is the template to send
     * Must be an integer that identifies a valid template. Templates are created in the admin tools.
-    
+
+* `personalisation` is the template to send
+    * Must be an dictionary, with keys matching the placeholders in the template.
+
```

Comment:
Updated the README in d10a943. It needs to be valid JSON, not just any old dict (which is why the single quotes didn’t work).